py_ecc/bls/hash.py

```diff
@@ -15,3 +18,39 @@ def hash_eth2(data: Union[bytes, bytearray]) -> Hash32:
     a future Ethereum 2.0 deployment phase.
     """
     return Hash32(hashlib.sha256(data).digest())
+
+
+def hkdf_extract(salt: Union[bytes, bytearray], ikm: Union[bytes, bytearray]) -> bytes:
+    """
+    HKDF-Expand
+
+    https://tools.ietf.org/html/rfc5869
+    """
+    return hmac.new(salt, ikm, hashlib.sha256).digest()
+
+
+def hkdf_expand(prk: Union[bytes, bytearray], info: Union[bytes, bytearray], length: int) -> bytes:
```

Comment:
You could use [`typing.ByteString`](https://docs.python.org/3/library/typing.html#typing.ByteString) in place of `Union[bytes, bytearray]` (though that also includes `memoryview` types which I think should be fine.

Reply:
Sounds good, I'm happy to switch it to `ByteString` and include `memoryview`

Reply:
I remember why I choose `Union[bytes, bytearray]` now and that is because the underlying `sha256` function takes `Union[bytes, bytearray]` so I will switch it back to `Union`